lib/Chunk.js

```diff
@@ -34,50 +75,104 @@ const getModulesIdent = set => {
 	return str;
 };
 
+/**
+ * @param {Set} set the set to convert to array
+ * @return {Array} the array returned from Array.from(set)
+ */
 const getArray = set => Array.from(set);
 
+/**
+ * @param {Set<Module>} set the Set to get the count/size of
+ * @return {number} the size of the modules
+ */
 const getModulesSize = set => {
-	let count = 0;
+	let size = 0;
 	for (const module of set) {
-		count += module.size();
+		size += module.size();
 	}
-	return count;
+	return size;
 };
 
+/**
+ * @class
```

Comment:
Lets remove extra `@class`, `@name` and `@description` directives for consistency.